README.md

```diff
@@ -4,7 +4,131 @@
 - [GoDoc](https://godoc.org/go.linecorp.com/centraldogma)
 - [CLI manual](https://line.github.io/centraldogma/client-cli.html)
 
-## How to build
+## Usage
+
+### Getting started
+
+```go
+import "go.linecorp.com/centraldogma"
+
+// create client with OAuth2 token
+centraldogma.NewClientWithToken(baseURL, token, nil)
+```
+
+### Customize transport
+
+If transport is `nil` (like above), `http2.Transport` is used by default. You could inject your own transport easily:
```

Comment:
How about splitting this line into two lines?

Reply:
Let me update it along with others comments :+1: